aws/aws-how-to/instances/upgrade-to-ubuntu-pro-at-scale-using-tokens-with-ssm.rst

```diff
@@ -0,0 +1,120 @@
+Upgrade to Ubuntu Pro at scale using tokens with SSM
+====================================================
+
+
+Overview
+---------
```

Comment:
RST is very picky about the length of these underlines matching the length of the word they underline. Please fix these so the CI jobs can pass.